options.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/getkin/kin-openapi/openapi3"
 	"github.com/golang-jwt/jwt/v5"
+	"github.com/rs/cors"
 )
```

Comment:
`rs/cors` is a middleware, I don't want to add it to fuego : users should be free to chose the middlewares they want (some users don't need cors)